### ISSUE TITLE
Add possibility to show baggage info

### ISSUE THREAD

**Issue by jaroslav-kubicek** [kiwicom/smart-faq#453]
Acceptance criteria:

- [x] When clicking on Baggage, specific FAQ category is shown
- [x] List of bought baggages is shown
- [x] "Add more baggage" redirects Baggage section in MMB
- [ ] Show info that no adiditional baggage can be added if that happens

<img width="551" alt="screen shot 2018-06-20 at 12 00 01" src="https://user-images.githubusercontent.com/2174629/41651711-7c8f0528-7481-11e8-8a83-b40fffbb6131.png">

Zeplin: https://zpl.io/bPYv4nk

**Comment by mjancik** [comment]
There has been a last minute update in the design based on a decision to show baggage to every passenger (to make it consistent with our booking page as well as new transactional emails). The zeplin link is updated. 
The difference is that the cards are expandable and show which passengers have the selected type of baggage. Try clicking it here: https://marvelapp.com/6hf9a7b

Also I added to zeplin a screen where all the possible states are available: https://zpl.io/aN09rwe

**Comment by jalbertsr** [comment]
Has it to be available also for past bookings?

**Comment by jalbertsr** [comment]
General notes about the issue:
This is not the final version, we are waiting for backend to implement some fixes and do some changes in the baggage structure, so we decided to better go with a working simple smaller version than waiting and not delivering anyhing for weeks. 

Missing parts: 
- Personal Items
- Possibility to know passanger name associated to one exact baggage.
- Extra and overweight baggage.

Test: https://kiwicom.github.io/smart-faq/

**Comment by MagdalenaAlmanova** [comment]
@jalbertsr Thank you for your comment below this post, it really helps me to save my time :) 

I found just minor things: 
1. The icon of baggage and the text "Your baggage" should be better centralized. See on the screenshot.
Demo v.
![screen shot 2018-06-28 at 22 00 06](https://user-images.githubusercontent.com/37298895/42057831-d3ac8224-7b1e-11e8-8914-8adfbf75931c.png)
Prototype v.
![screen shot 2018-06-28 at 22 00 18](https://user-images.githubusercontent.com/37298895/42057836-d92d0d9a-7b1e-11e8-97ee-d9bfdb0ec84c.png)

2. The text "Here is your baggage allowance for the trip" should have a different color of the font. At the end of a sentence there should be a dot (.), I think. 
3. Is it correct when there is a missing quantity of baggage? 
demo v.
![screen shot 2018-06-28 at 22 04 35](https://user-images.githubusercontent.com/37298895/42058005-4e670796-7b1f-11e8-9836-1a05d9f384b5.png)
prototype v.
![screen shot 2018-06-28 at 22 04 48](https://user-images.githubusercontent.com/37298895/42058030-581b072e-7b1f-11e8-9379-6f3e6585b5ed.png)
4. The size of the baggage should be slightly lower and there is missing "cm". Shouldn't the size of the font be bigger?
demo v.
![screen shot 2018-06-28 at 22 07 10](https://user-images.githubusercontent.com/37298895/42058131-ba68fabc-7b1f-11e8-9c6a-6ed30557c27d.png)
Prototype v.
![screen shot 2018-06-28 at 22 07 35](https://user-images.githubusercontent.com/37298895/42058136-c08e43ca-7b1f-11e8-8614-eb69b6bb5813.png)

**Comment by MRTNKCR** [comment]
Is this issue wiating for a QA please @MagdalenaAlmanova ?

**Comment by MagdalenaAlmanova** [comment]
No, I think that @jalbertsr can fix mistakes, which I found. I am constantly checking this, whether this was already fixed. You can see that, there is an open issue 507 below your comment. :)

**Comment by MRTNKCR** [comment]
@MagdalenaAlmanova Ok, so why is the issue in QA column then? If there is some dev work missing, move it to back `In progress` please.

**Comment by kantord** [comment]
@MRTNKCR I reply because @jalbertsr is on a hackathon, and I don't want to make you wait for the info. We decided this together with @jalbertsr. 

If it doesn't meet the quality standards, I say we should disable this so the users don't see it until it's not ready. (Not rollback, because that would sabotage work on this issue: https://github.com/kiwicom/smart-faq/issues/454)

We pair programmed with @jalbertsr on this issue, so I'm quite familiar with it, plus I'm already working on the same parts of code for https://github.com/kiwicom/smart-faq/issues/454 so I can step in to make fixes on this while @jalbertsr is on the hackathon.

**Comment by MRTNKCR** [comment]
Ok, undestand, thank you for the explanation! I hope you understand my point - it's not that much about quality standards, but we should either display all bags the user has in his booking or none of them. Displaying just some of them can be confusing and might drive even more contacts on CS.

**Comment by kantord** [comment]
Regarding the displaying of special luggage (i.e. stuff that's missing from the APIs currently) I think we have two options:
- wait with this until it becomes available (:disappointed:); or
- add a text under the baggage list saying something like ("Special baggage items such as extra hand luggage or sports equipment are not included here")

**Comment by kantord** [comment]
> 4. [..] Shouldn't the size of the font be bigger?

All font sizes seem to be correct to me @MagdalenaAlmanova

**Comment by kantord** [comment]
Another issue I found here is that the "Add more baggage" link is missing.

**Comment by kantord** [comment]
The rest of the issues I fixed here https://github.com/kiwicom/smart-faq/pull/516/files

**Comment by MRTNKCR** [comment]
@kantord Ok, great idea, let's display the fallback info and add the special baggage data later.

**Comment by MRTNKCR** [comment]
@mjancik Can you please with the proper copy? Thank you ✌️

**Comment by mjancik** [comment]
So what are the exact cases of baggage we are not going to show? Should we communicate that they can check it in MMB? Isn't it less confusing to just show a Description and button which would say: "See your baggage" and it would open MMB with the baggage section?

**Comment by MRTNKCR** [comment]
Missing parts:

- Personal Items
- Possibility to know passanger name associated to one exact baggage.
- Extra and overweight baggage.

Let's stick with displaying the standard luggage + the additional info saying these types of luggage are not being displayed. It will be there just for a short time hopefully.

**Comment by kantord** [comment]
> Let's stick with displaying the standard luggage + the additional info saying these types of luggage are not being displayed. It will be there just for a short time hopefully.

@mjancik what text should be displayed and how should it look? :-)

**Comment by MRTNKCR** [comment]
@kantord the thing is in progress, Martin will deliver it soon :-)

**Comment by kantord** [comment]
:heart:

**Comment by MRTNKCR** [comment]
@kantord here it is - the MVP section. The copy is yet to by finalized. https://app.zeplin.io/project/5a8e8992554d385c76ad5aeb/screen/5b2a732a2244219b1ad96f2d

**Comment by mjancik** [comment]
So I have received the copy changes. The copywriters where suggesting a change in the layout. I changed it in zeplin, so please pay attention to it. Thank You.

**Comment by kantord** [comment]
pls after QA create new issues for the missing stuff

**Comment by MagdalenaAlmanova** [comment]
Is it correct when the button "more info" does not work? (the redirecting website isn't available) 
Something that I noticed: When I click on "baggage" or "boarding pass" it shows me baggage or boarding passes, but when I "unclicked" "baggage" or "boarding pass" it shows me nothing. I think that it should show FAQ categories. or? 
![screen shot 2018-07-05 at 12 39 36](https://user-images.githubusercontent.com/37298895/42318835-8664060e-8050-11e8-9a9a-0a9c2a537747.png)
@kantord  @vborszeki 

Thanks

**Comment by vborszeki** [comment]
@MagdalenaAlmanova Yes, it's valid, that you don't see the page by clicking the 'More Info' as it should open a new page, which is outside our app. Toggling back the 'Baggage' and 'Boarding passes' should not make disappear the FAQ cards.

**Comment by MagdalenaAlmanova** [comment]
Ok so, I reopen this issue again. :/

**Comment by kantord** [comment]
@MagdalenaAlmanova what happens for you when you press "More info"? I think it doesn't work because Manage My Booking is not available on the domain you're testing on :(

**Comment by MagdalenaAlmanova** [comment]
@kantord I already know about it. But I reopen this issue because `When I click on "baggage" or "boarding pass" it shows me baggage or boarding passes, but when I "unclicked" "baggage" or "boarding pass" it shows me nothing. I think that it should show FAQ categories. `
![screen shot 2018-07-08 at 21 36 05](https://user-images.githubusercontent.com/37298895/42423216-1f8cf1bc-82f7-11e8-9a39-b2d85c6381cf.png)
This bug is still persistent...

**Comment by MRTNKCR** [comment]
@mjancik Can you please describe how it should work with clicking/unclicking the Bag button? Please also cover the scenario where I click the button, navigate to a different category and then unclick it. Thank you ✌️

**Comment by mjancik** [comment]
I have already explained in #454

**Comment by kantord** [comment]
GraphQL query ready for the new baggage data. Waiting for the stable version of the API to be released.
 There is a bug with email based bookings on the backend now, which inhibits us from testing this with SmartFAQ, but once that's fixed we can start implementing this in SmartFAQ.

**Comment by MRTNKCR** [comment]
@jaroslav-kubicek @jalbertsr guys, why is this feature somewhere deep down in `TO DO`??? It's supposed to be almost ready! Who's pushing it out when it's totally invisible here?? What's missing? The proper API endpoint? Anything else? Can you pls provide the analytics spec so that we have it in place before the release @mjancik ? Thanks ✌️

**Comment by jaroslav-kubicek** [comment]
It was in TODO since @kantord  was waiting for proper API, the only struggle is that someone needs to take care of it now instead... 😢

**Comment by MRTNKCR** [comment]
Please, can you ask Natali to define the metrics for this feature? @mjancik  I'd like to have the analytics in please before the release. Thank you.

**Comment by kantord** [comment]
Created separate issue: https://github.com/kiwicom/smart-faq/issues/706
